jobs/geo/core_us.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import annotations
```

Comment:
I'm not sure I agree with breaking the "US" ones into a separate file. There is enough in common between this and the CA tiger files that I think they could be loaded using the same script.

Reply:
I did this to address the `PLR0915 Too many statements` error I was getting. Only other way would be to modify max statements and go against Pep8. Unless you know of a 3rd option? @ian-r-rose

Reply:
Perhaps if I remove the line of code that adds the year to the table (that we reference below) that will reduce the statements and address the issue.

Reply:
This rule is intended to encourage shorter *functions* rather than files. So you could address the error by breaking it into logical functions. But if you implement some of my other suggestions, I think the function should wind up shorter than it currently is.

Ultimately, I'm not a huge fan of `PLR0915 Too many statements`, so we could also just disable that rule :)

Reply:
done